kratos/tests/cpp_tests/strategies/strategies/test_strategies.cpp

```diff
@@ -40,7 +40,7 @@
 #include "solving_strategies/convergencecriterias/residual_criteria.h"
 
 // The most builder and solver (the block builder and solver)
-#include "solving_strategies/convergencecriterias/residual_criteria.h"
+#include "solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h"
```

Comment:
is this intentional?

Reply:
I will check, after solving conflicts who knows

Reply:
ping